spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/SampleControllerHelperService.java

```diff
@@ -767,8 +786,9 @@ public JSONObject listSamplesDataTable(HttpSession session, JSONObject json) {
                       sample.getSampleType() + "','" +
                       (sample.getQcPassed() != null ? sample.getQcPassed().toString() : "") + "','" +
                       getSampleLastQC(sample.getId()) + "','" +
-                      "<a href=\"/miso/sample/" + sample.getId() + "\"><span class=\"ui-icon ui-icon-pencil\"></span></a>" + "']");
-
+                      "<a href=\"/miso/sample/" + sample.getId() + "\"><span class=\"ui-icon ui-icon-pencil\"></span></a>" + "','" +
+                      (sample.getIdentificationBarcode() != null ? sample.getIdentificationBarcode() : "") + "','" +
+                      "']");
```

Comment:
Passing HTML back from the controller - :crying_cat_face:   Although, it seems to be what has always been done here.  I'll make a note to review.  :white_check_mark: